README.md

```diff
@@ -3,13 +3,41 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  1.Dropping constants and lower order terms 2.Input size:\
+  \
+  We drop and disregard the lower order terms when doing asymptotic analysis and these could have actual impact on performance in real practice. So let's say we have $3n^3 + 2n^2 + 50n + 100$. When you do asymptotic analysis we are only focusing on the term with the highest growth rate. $3n^3$ has the highest growth rate. We would disregard the lower order terms and constant giving us a complexity of $O(n^3)$. This is good for large input sizes but the lower order terms and the constant could effect this with smaller input sizes. If this algorithm had a constant factor of 3 we would have $3n^3$ and we have a second algorithm that's $O(n^2)$ with a constant factor of 100.\
+    1st. $3(10^3) = 3000$\
+    2nd. $100(10^2) = 10,000$\
+  So the first algorithm is faster with a smaller input size, if we make the first input size the same as the second n=100, then we see the $n^3$ grows much faster and becomes slower then the 2nd algorithm. This is why input size matters and why the complexity can show the scaling of an algorithm but disregarding the constant factors can be misleading. An algorithm with a lower asymptotic complexity can be slower if it has a larger constant factor.
+     
+
+2. In asymptotic analiysis we treat all constants at the same rate. When in practice constants can take different amounts of time. Like comparing strings takes longer then comparing integers.
```

Comment:
That sounds the same as your reason above.

Reply:
Still relevant -- and if constants take different amounts of time, they can't be constant.